components/tyk-dashboard/templates/secret-dashboard.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 type: Opaque
 data:
   {{ if .Values.global.adminUser.password }}
-  TYK_PASS: {{ .Values.global.adminUser.password | b64enc | quote }}
+  adminUserPassword: {{ .Values.global.adminUser.password | b64enc | quote }}
```

Comment:
Don't we need to check if useSecretName is set?

Reply:
Agreed.
I am saying if I will set value of  admin credentials using `global.adminUser.useSecretName`, field in above secret file will still use the value from {{.Values.global.adminUser}} instead of what is set in `global.adminUser.useSecretName`

Reply:
I see. I updated the PR accordingly. What do you think now?

The secret will be created if `global.adminUser.useSecretName` is not in use. We refer to this secret in bootstrap template, [here](https://github.com/TykTechnologies/tyk-charts/blob/46da947bc88204c36f04e58ed258a498e3727abb/components/tyk-bootstrap/templates/bootstrap-post-install.yaml#L68). The usage of adminUser secret is checked here also.

https://github.com/TykTechnologies/tyk-charts/pull/103/commits/46da947bc88204c36f04e58ed258a498e3727abb#diff-47a90fa50b7ecca7234fdb3246886e73483d76ef986b5d74bf5d2a5ff151ed56R1

Reply:
Looks great! Thank you